pytest_marker_bugzilla.py

```diff
@@ -312,10 +320,17 @@ def pytest_configure(config):
         "markers",
         "bugzilla(*bug_ids, **guards): Bugzilla integration",
     )
+    
     if config.getvalue("bugzilla") and config.getvalue('bugzilla_url'):
         url = config.getvalue('bugzilla_url')
-        user = config.getvalue('bugzilla_username')
-        password = config.getvalue('bugzilla_password')
+        if config.getvalue('bugzilla_username') and config.getvalue('bugzilla_password'):
+            user = config.getvalue('bugzilla_username')
+            password = config.getvalue('bugzilla_password')
+            bz = bugzilla.Bugzilla(url=url, user=user, password=password)
+        elif config.getvalue('bugzilla_api_key'):
+            api_key = config.getvalue('bugzilla_api_key')
+            bz = bugzilla.Bugzilla(url=url, api_key=api_key)
+    
```

Comment:
in case both conditions are false, plugin will fail later on line 342, on undefined `bz` variable

Reply:
Right, it should be handled now by:
https://github.com/EricLSD/pytest_marker_bugzilla/pull/47/commits/a1705f984779803c55f3d2482560400a56559a32